database/migrations/default/1746807631787_patient_inj_sev_update/up.sql

```diff
@@ -0,0 +1,258 @@
+alter table ems__incidents drop column patient_injry_sev_id;
+
+alter table ems__incidents
+add column patient_injry_sev_id integer generated always as (
+    case
+        --
+        -- fatal injuries
+        --
+        when
+            (lower(pcr_provider_impression_primary) = 'death on scene')
+            or
+            (lower(pcr_provider_impression_secondary) = 'death on scene')
+            or
+            (lower(pcr_outcome) = 'deceased on scene')
+            or
+            (
+                lower(pcr_patient_acuity_final)
+                = 'dead without resuscitation efforts (black)'
+            )
+            or (
+                (
+                    lower(pcr_patient_acuity_initial)
+                    = 'dead without resuscitation efforts (black)'
+                )
+                and (
+                    pcr_patient_acuity_final is NULL
+                )
+            )
+            then 4
```

Comment:
this block now checks two fields we weren't previously looking at:
* `pcr_patient_acuity_initial` (when `pcr_patient_acuity_final` is null)
* `pcr_provider_impression_secondary`

Reply:
My first thought was to give users the ability to manually edit the injury status, but i think lets avoid that until xavier makes it known that is a need if weird cases like that keep stacking up? for now using the higher injury severity when a fatality is involved seems okay